librosa/decompose.py

```diff
@@ -322,7 +357,7 @@ def hpss(S, kernel_size=31, power=2.0, mask=False):
         perc[zero_harm & zero_perc] = 0.5
 
         # Compute harmonic mask
-        mask_harm = harm / (harm + perc)
-        mask_perc = perc / (harm + perc)
+        mask_harm = (harm >  perc * margin_harm).astype(float)
```

Comment:
Hex @Cortexelus,

you know about the TSM toolbox from Jonathan?
He's following a similar strategy: PV for harmonic signal and Overlap-Add for transients.
I think that's also what the rubberband people are doing.

https://www.audiolabs-erlangen.de/resources/MIR/TSMtoolbox/